Gemfile

```diff
@@ -51,6 +51,7 @@ group :development, :test do
   gem "awesome_print"
   gem "guard-rspec"
   gem "pry-rails"
+  gem "pry-nav"
```

Comment:
Why didn't I know about this gem! :)

Reply:
I always forget which gems you need to add to make `pry` function normally. This should really be part of `pry-rails`